.env

```diff
@@ -0,0 +1,24 @@
+# common settings
+TZ=Asia/Shanghai
+NETWORKS_DRIVER=bridge
+
+
+# resource limit
+LIMIT_MEMORY_MYSQL=2G
+LIMIT_MEMORY_REDIS=2G
+
+
+# PATHS
+MySQL_PATH=./data/mysql # mysql data storage path
+REDIS_PATH=./data/redis # redis data storage path
+
+
+# MYSQL
+MYSQL_PORT=3306
+MYSQL_USERNAME=admin
+MYSQL_PASSWORD=123456
+MYSQL_ROOT_PASSWORD=123456
+MYSQL_DATABASE=dev-database # database name
```

Comment:
Can you change it to `DATABASE_URL` and make it default to a local sqlite3 file?

Reply:
Should sqlite or mysql be used based on the backend environment variables in settings.py?

Reply:
The `settings.py` should also read from `DATABASE_URL` if present.
Appreciated if you also change it.

Reply:
submitted.

Reply:
My opinion is, this docker compose file should be the configuration to run in a production environment, not sqlite3, development environments can be used as needed.

Reply:
> My opinion is this docker compose file should be the configuration to run in a production environment, not sqlite3, development environments can be used as needed.

Agree on that, my concern is it exposes way too many env vars for users to specify, I would suggest the following:

```
DATABASE_URL=mysql://admin:123456@localhost:3306/dev
```
And rename the `MYSQL_PATH` to `DATA_PATH` and `MYSQL_PORT` to `DB_PORT`

Because most DBaaS expose an environment variable with the connection string, which can be also recognized by SQLAlchemy. And it reduces the total number of env vars users need to specify. In fact, I also prefer PostgreSQL since it's the choice of many cloud services.

With this change, we unify the ways to specify DB in both container and locally. It can point to a sqllte3 file for local development.

Reply:
> In fact, I also prefer PostgreSQL

I also want to try using PostgreSQL. I will modify it and push it later.

Reply:
> Agree on that, my concern is it exposes way too many env vars for users to specify, I would suggest the following:
> 
> ```
> DATABASE_URL=mysql://admin:123456@localhost:3306/dev
> ```

Configuration via DSN is not supported in MySQL/PostgreSQL base image. 
According to the current investigation, it may be necessary to rebuild the base image. But I don't think this is the right solution. Do you have any better suggestions?

> And rename the `MYSQL_PATH` to `DATA_PATH` and `MYSQL_PORT` to `DB_PORT`

This is a good suggestion, I have modified it!

Reply:
> Configuration via DSN is not supported in MySQL/PostgreSQL base image. According to the current investigation, it may be necessary to rebuild the base image. But I don't think this is the right solution.

I am fine with the current way, but use Postgres instead.